workflow/snakemake_rules/nextclade.smk

```diff
@@ -241,17 +257,17 @@ rule nextclade_info:
 
 
 rule combine_alignments:
-    message:
-        """
-        Generating full alignment by combining newly aligned sequences with previous (cached) alignment
-        """
+    """
+    Generating full alignment by combining newly aligned sequences with previous (cached) alignment
+    """
     input:
-        old_alignment=f"data/{database}/nextclade.aligned.old.fasta",
-        new_alignment=f"data/{database}/nextclade.aligned.upd.fasta",
+        nextclade_finished=f"data/{database}/nextclade.touch"
```

Comment:
```suggestion
        nextclade_finished=f"data/{database}/nextclade.touch",
```

Reply:
Yes that's right, we only use the tsv from the 21L build. I'm not sure that this is causing complexity beyond having an extra wildcard `{reference}` but I'll see whether splitting it up into main Nextclade and 21L Nextclade is easier to read.